### ISSUE TITLE
Extendable Functional and Stable UI

### ISSUE THREAD

**Issue by shahrukhx01** [lalitpagaria/obsei#147]
@lalitpagaria **Is your feature request related to a problem? Please describe.**
Have extendable and standard UI for the drag and drop interface.

**Describe the solution you'd like**
Write a small wrapper on top of https://justpy.io/ and create components for most generic items, for instance, node, graph, link, etc. I think using canvas element will help in doing this. It'd be possible without canvas too, need to explore little more.

**Comment by shahrukhx01** [comment]
@lalitpagaria you can assign me this.

**Comment by NirantK** [comment]
Would it be worthwhile to have demo "applets" which would integrate with more low code tools like Streamlit, Retool, and AppSmith? 

This approach could help Obsei get more users without waiting for a fully functional/extensible UI early on.

**Comment by shahrukhx01** [comment]
@NirantK We already have a basic UI in streamlit I believe, we were planning to try some other framework where we can add drag and drop feature. @lalitpagaria can comment more on this.

**Comment by lalitpagaria** [comment]
Yes @NirantK  we already hai streamlit based UI. You can spin up locally.
Please instructions out at https://www.github.com/obsei/obsei/tree/master/sample-ui%2FREADME.md

**Comment by NirantK** [comment]
Not in the sense of using Obsei, but for showing the final output or a summary of the same. E.g. in the example above, Slack is where you finally publish output. 

But nevertheless, this shows that it's easy enough for me to build a Streamlit app for this. I'll give it a shot when I get there.

**Comment by shahrukhx01** [comment]
For that we also had a discussion around, we have a separate repo for that, I have added some details for that, if that's interest, you can take it up and I & Lalit will try our best to assist you. Following is the repo:
https://github.com/lalitpagaria/obsei-concept

**Comment by lalitpagaria** [comment]
@NirantK Yes we are working on creating more final output.
One I captured in this Demo video (Twitter->Anonymizer->Slack): https://www.youtube.com/watch?reload=9&v=GTF-Hy96gvY
Check this out.
